trace/src/main/org/jetbrains/lincheck/trace/Serialization.kt

```diff
@@ -10,14 +10,24 @@
 
 package org.jetbrains.lincheck.trace
 
+import org.jetbrains.kotlinx.lincheck.trace.ClassDescriptor
+import org.jetbrains.kotlinx.lincheck.trace.CodeLocations
+import org.jetbrains.kotlinx.lincheck.trace.FieldDescriptor
+import org.jetbrains.kotlinx.lincheck.trace.IndexedPool
+import org.jetbrains.kotlinx.lincheck.trace.MethodDescriptor
+import org.jetbrains.kotlinx.lincheck.trace.MethodSignature
+import org.jetbrains.kotlinx.lincheck.trace.TRACE_CONTEXT
+import org.jetbrains.kotlinx.lincheck.trace.TraceContext
+import org.jetbrains.kotlinx.lincheck.trace.Types
+import org.jetbrains.kotlinx.lincheck.trace.VariableDescriptor
 import java.io.*
 
 private const val OUTPUT_BUFFER_SIZE: Int = 16*1024*1024
 
 const val TRACE_MAGIC : Long = 0x706e547124ee5f70L
 const val TRACE_VERSION : Long = 6
 
-fun saveRecorderTrace(out: OutputStream, context: TraceContext, rootCallsPerThread: List<TRTracePoint>) {
+internal fun saveRecorderTrace(out: OutputStream, context: TraceContext, rootCallsPerThread: List<TRTracePoint>) {
```

Comment:
I think these should also be kept public. 
Please rollback all `internal` changes in this module.